extensions/spring-xd-extension-kafka/src/main/java/org/springframework/integration/x/kafka/KafkaSourceModuleOptionsMetadata.java

```diff
@@ -31,7 +36,7 @@
 @Mixin({KafkaZKOptionMixin.class, KafkaConsumerOptionsMixin.class, KafkaOffsetTopicOptionsMixin.class})
 public class KafkaSourceModuleOptionsMetadata implements ProfileNamesProvider {
 
-	private String topic = ModulePlaceholders.XD_STREAM_NAME;
+	private String topics = ModulePlaceholders.XD_STREAM_NAME;
```

Comment:
I'm wondering if we shouldn't support both options (`topic`) and (`topics`) and allow only one to be set - for backwards compatibility - e.g. for existing deployments, etc. Granted, this adds some complexity ...